host/xtest/regression_5000.c

```diff
@@ -541,7 +555,7 @@ static void Register_sharedMemory_aboveMaxSize(struct xtest_session *cs)
 	Do_ADBG_BeginSubCase(cs->c, "Register_sharedMemory_aboveMaxSize");
 	{
 		TEEC_Result res;
-		uint32_t size_aboveMax = 0xffffffff;
+		uint32_t size_aboveMax = UINT32_MAX;
```

Comment:
This should be a `size_t` and the value `SIZE_MAX`.
`RegisterSharedMemory()` will need to be updated to take the size as `size_t` instead of `uint32_t` (that's what we get for using fixed sized integers where they don't belong).